.gitignore

```diff
@@ -191,6 +191,6 @@ Temporary Items
 .apdisk
 
 # Clinica specific files
-
+clinica/resources/masks/*
```

Comment:
Do you remember what generates these?

Reply:
Yes I installed clinica locally with `pip install -e .` (it wasn't working with poetry)
And then when I ran the pipeline with my installation the masks were downloaded there.

Reply:
Ok, you did well to exclude them here then 👍